components/brave_shields/core/common/features.cc

```diff
@@ -92,11 +92,11 @@ BASE_FEATURE(kBraveReduceLanguage,
 // When enabled, brave shred feature will be available
 BASE_FEATURE(kBraveShredFeature,
              "BraveShredFeature",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 // When enabled, brave shred will clear all cache data when shredding.
 BASE_FEATURE(kBraveShredCacheData,
              "BraveShredCacheData",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
```

Comment:
Won't this enable it for all platforms?

Reply:
The feature doesn't exist on other platforms, but I should use build flag so it's iOS specific so the flag can be re-used by other platforms later.